grouper/fe/handlers.py

```diff
@@ -841,6 +842,40 @@ def post(self, group_id=None, name=None):
         return self.redirect("/groups/{}".format(group.name))
 
 
+class GroupRemove(GrouperHandler):
+    def post(self, group_id=None, name=None):
+        group = Group.get(self.session, group_id, name)
+        if not group:
+            return self.notfound()
+
+        members = group.my_members()
+        if self.current_user.my_role(members) != "owner":
```

Comment:
Managers see remove buttons for members, but when they attempt to remove a member, their efforts fail here.